src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -3044,6 +3044,131 @@ Many use cases can be fulfilled using this API. For instance:
   can scale up and down these resources when receiving the corresponding signals from users (via the Workflow Execution portal),
   or from third-party systems (via the Signals REST endpoint).
 
+== Third party services API
+
+Some ProActive services offers java Clients ready to be used in ProActive's workflows.
```

Comment:
In addition to the core component APIs described above, ProActive worfklow tasks can also communicate with other  ProActive Modules such as Service Automation and Notification Service.